Tiltfile

```diff
@@ -39,7 +39,10 @@ config.define_string("webHost", False, "Public hostname for port forwards")
 
 # Components
 config.define_bool("algorand", False, "Enable Algorand component")
+config.define_bool("eth2", False, "Enable second Eth component")
```

Comment:
Normally names, don't matter, but eth2 literally is the name of the ethereum beacon chain that is going to be the "consensus layer" post-merge.

I'd suggest finding a better name because this is confusing a bit.

[An overview of eth2](https://www.gfinityesports.com/cryptocurrency/ethereum-2-release-date-eth2-roadmap-phases-is-ethereum-2-new-coin-serenity/)

It is a bit more unweildy, but perhaps: `second_eth`

Reply:
agreed, this made me uncomfortable for the same reason the whole time I was coding it (like that's not what eth2 means). I'll change it to `second_eth` even though it sounds dumb 😛